src/lib/numbers.js

```diff
@@ -0,0 +1,57 @@
+const add = (a, b) => {
+  return a + b;
+};
+
+const subtract = (a, b) => {
+  return a - b;
+};
+
+const multiply = (a, b) => {
+  return a * b;
+};
+
+const divide = (a, b) => {
+  return a / b;
+};
+
+const power = (a, b) => {
+  return Math.pow(a, b);
+};
+
+const round = a => {
+  return Math.round(a);
+};
+
+const roundUp = a => {
+  return Math.ceil(a);
+};
+
+const roundDown = a => {
+  return Math.floor(a);
+};
+
+const absolute = a => {
+  return Math.abs(a);
+};
+
+const quotient = (a, b) => {
+  return ~~(a / b);
```

Comment:
Eslint is throwing an error here. As it happens theres a built in function for this: Math.trunc (see [docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc#:~:text=The%20Math.,by%20removing%20any%20fractional%20digits.))
```
  return Math.trunc(a / b);
```